Userland/Libraries/LibCpp/Preprocessor.h

```diff
@@ -14,6 +14,14 @@
 #include <AK/Vector.h>
 
 namespace Cpp {
+
+struct TodoEntry {
```

Comment:
Does the `TodoEntry` struct belong in `Preprocessor.h`?
I think a better place to define it in would be in `Parser.h`, just before the `get_todo_entries()` function.

Reply:
Done :)